src/sdk/entry.ts

```diff
@@ -37,9 +40,11 @@ if (customHostBaseUrl && isValidCustomHostUrl(customHostBaseUrl)) {
 
 // Setting LP if enabled
 ContentstackLivePreview.init({
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-//@ts-ignore
-stackSdk:Stack,
+  //@ts-ignore
```

Comment:
ts-ignore is required here ? as in other prs I saw it getting removed ? @Amitkanswal

Reply:
yup, but by doing so LP was getting affected. So I undo my changes. I have shared this feedback to the ghost team and they will update LP typescript in the future release.